packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1134,12 +1134,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.handleContext,
             blobManagerSnapshot,
             () => this.storage,
-            (blobId, localId) => {
+            (localId: string, blobId?: string) => {
                 if (!this.disposed) {
-                    this.submit(ContainerMessageType.BlobAttach, undefined, undefined, { blobId, localId });
+                    this.submit(ContainerMessageType.BlobAttach, undefined, undefined, { localId, blobId });
```

Comment:
Out of curiosity, this change here didn't need to be made right? This is a change more for consistency's sake than anything.

Reply:
Yes, that's right :-).